packages/components/inputs/rich-text-utils/src/rich-text-body/rich-text-body.tsx

```diff
@@ -87,23 +143,18 @@ const Item = styled.div`
   text-align: left;
 `;
 
-const StylesDropdownItem = (props) => {
-  const as =
-    Object.keys(BLOCK_TAGS).find((key) => BLOCK_TAGS[key] === props.value) ||
-    'div';
+const StylesDropdownItem = (props: TStylesDropdownItem) => {
+  const asProp = (Object.keys(BLOCK_TAGS).find(
+    (key) => BLOCK_TAGS[key as keyof typeof BLOCK_TAGS] === props.value
+  ) || 'div') as React.ElementType;
```

Comment:
nit: Add `ElementType` to the rest of React imports at the beginning of the file.